### ISSUE TITLE
Manage expectations of Minetest Game

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest_game#2555]
According to paramat, MTG is not a game:

> MTgame is not meant to be a complete game, it is a basic sandbox meant to be modded.

https://forum.minetest.net/viewtopic.php?p=364736#p364736

However, the way how MTG is currently presented to people gives a strong impression that MTG is a real game (instead of a “modding base”).

These official resources are affected:

* README
* Minetest Wiki
* minetest.net homepage
* possibly others?

**Comment by SmallJoker** [comment]
Different games, different concepts. Minetest Game is a game but has different objectives than survival-focussed games.
Is it wrong to have a game which provides a common modding base?

**Comment by sfan5** [comment]
> According to paramat, MTG is not a game:

But that's not what the quote says, what it says is:

> According to paramat, MTG is not a game <i>that is complete</i>:

**Comment by Fixer-007** [comment]
Just add damn mobs to it and it will be finally a bare game.

**Comment by Wuzzy2** [comment]
Sorry, my initial request came over as harsh. I was in a hurry, and I did not mean to be harsh. The point of this issue is not to bash MTG. Or to request any new features. It's about something completely different.

My problem is how Minetest Game is presented to people. Many many people will expect MTG to be something that it is not, but in official resources it's easy to get a false impression. This is a problem because this leads to frequent disappointment. And this is bad, because to many people, MTG = MT (even if this is completely false), so a bad impression of MTG will be the same as a bad impression of MT. Bad presentation of MTG is harming MT as well.

A big problem is the name “Minetest Game”. “Game” suggests gameplay and yes, MTG does have some gameplay, but it is (sorry) rather shallow and I think that's not the actual goal anyway. But I fear a name change is not going to happen anytime soon. So yeah, MTG is *technically* the game but it's very clear that the main focus is something entirely different.

It's better to focus on what MTG is *actually* good at. MTG is good in combination with mods and when used on *servers*, when on said servers it is combined with mods. MTG might also be interesting if you want to understand how the modding infrastrucutre works in general, but I'm not sure about this.
Anyhow, when MTG is combined with the correct mods, you can get a pretty decent experience.

But the bare MTG without any mods is too incomplete for my taste. I understand the quote as this is by design. Yet many many people will expect MTG to be a “full” game.

Now, back to the main point:
Currently, there is just no real good way how MTG is presented to newcomers. The actual strengths of MTG, namely, moddability and usage on servers, are not used as a “selling point”. Instead, you get the impression that MTG is meant as a full, complete **standalone** game, which is probably not intended.

Let's figure out *how* MTG could be “advertised”, described so that newcomers know what it is *really* about, to avoid a wrong impression on MT and to avoid disappointment.
I hope that, when newcomers understand that MTG is NOT meant as a (full) game, *and* that there are alternatives, the reputation of MT might improve overall.

Thinking about this, I also notice another problem: MTG is best for *advanced users*. Why do I say this? First, there is no crafting guide, and the game explains nothing. If you do not know the wiki exists, you would be completely stumped, as you don't even know how to craft a pickaxe. Second, once you “mastered” MTG, it gets boring pretty fast, so there's a strong reason to check out mods. That's another advanced task (thankfully, not THAT advanced anymore thanks to ContentDB, but finding good mods still takes time). And lastly, MTG is probably best used on servers, which is definitely advanced use.
Therefore, I wonder if MTG is just presented to the completely wrong people. MTG is for people with a “build-your-own-game” mindset (installing lots of mods), and for server operators. If you expect MTG to be a complete game that is fun out of the box, you are the completely wrong audience. Yet MTG seems to try to capture people that expect a full game.

I do not have solutions ready, I need to think about this myself as well.

**Comment by paramat** [comment]
I agree that it needs to be made clear what MTGame is, so that players do not have high expectations for it.
I think something needs to be done because it seems many new players play unmodded MTGame and judge it against MC, and then become disappointed.

The title of this issue really needs changing, i almost closed the issue based on the title =) Maybe something mentioning the 'managing of expectations'.

**Comment by LoneWolfHT** [comment]
Re-doing the main menu so that it doesn't matter which games are bundled with MT (In the sense that it's just as easy to see the games out there as it is to see the games that are currently installed) needs to be given a little higher priority IMO

**Comment by unixuser9000** [comment]
MTG is most certainly NOT a game. This is a big issue that needs to be addressed in some way. If the developers of MTG are not willing to develop it further into a real game, it needs to stop being advertised as such.

**Comment by Moth-Tolias** [comment]
off topic [is there any better place to discuss this?] but i'm curious as to what a "complete" minetest_game would look like.

**Comment by paramat** [comment]
'Complete' is subjective, and i did write 'in a way complete'. I also wrote "When i write 'not a complete game' i am referring to common, mainstream and high expectations.".
> If the developers of MTG are not willing to develop it further into a complete game

It is intended to make it 'more complete', see the proposals in the issues, we just have very little time for MTG and progress is slow.
> it needs to stop being advertised as such.

It is not advertised as 'complete', but that misunderstanding often occurs, which is why i agree more clarity is needed.

The title of this issue and the first post are poorly worded, see https://github.com/minetest/minetest_game/issues/2555#issuecomment-570925154 for much better wording, which i agree with.

**Comment by unixuser9000** [comment]
I think that if MTG is not expected to be complete for a long time, it might be wise to add a "mod store" or a "game store" to the Minetest client, or at the very least make installing mods easier for beginners

**Comment by paramat** [comment]
That is already done in the Content Database.

Seems to me many players download and play without looking at websites or documentation, so clarifying of MTG may need to be in-game, maybe a tooltip or something.

**Comment by paramat** [comment]
The game 'description' needs changing:
https://github.com/minetest/minetest_game/blob/d03d7e9bdc4ab3325b632ed22338c1ab2c9b0a5c/game.conf#L3
This is the description that appears in the Content Database so is the critical place to manage expectations.
I think it must mention that there are no animals or monsters (not counting fixed-in-place insects), and that most players will need to add mods to create satisfying gameplay.

**Comment by paramat** [comment]
Initial suggestion for a new description:

"A basic exploration, mining, crafting and building sandbox game with no animals or monsters. Many users will want to add mods for satisfactory gameplay. Reliably maintained by Minetest core developers."

I removed the mention of being bundled with MT because hopefully that will not be the case soon, and if it is bundled that will already be obvious.
I removed the mention of 'lightweight' because although it somewhat is, it has excessive and poorly designed ABM use and so i feel does not deserve to be described as 'good lighweight design' anymore. Those who want a lightweight game should be encouraged to go elsewhere and choose a truly lightweight and better designed game.
I removed the mention of 'moddable' as that might encourage users to consider this a well-designed base for modding, which it is not, even though it is currently used that way a lot.
I removed the mention of being 'fairly playable without mods' as many users will not agree, and this sets expectations too high.

Ha ... so i removed everything in the current description =D
